bc_obps/service/tests/data_access_service/test_data_access_operation_designated_operator_timeline.py

```diff
@@ -22,25 +21,31 @@ def test_unapproved_user_exception():
     def test_get_operations_for_industry_user():
         approved_user_operator = baker.make_recipe('registration.tests.utils.approved_user_operator')
 
-        # transferred operation - should not be returned
+        # simulating a transferred operation - should not be returned
         baker.make_recipe(
             'registration.tests.utils.operation_designated_operator_timeline',
-            status=OperationDesignatedOperatorTimeline.Statuses.TRANSFERRED,
+            end_date="2024-12-25 01:00:00-08",
             operator=approved_user_operator.operator,
         )
 
-        # closed operations
-        baker.make_recipe(
-            'registration.tests.utils.operation_designated_operator_timeline',
-            status=OperationDesignatedOperatorTimeline.Statuses.CLOSED,
+        # active, registered operations
+        registered_operations = baker.make_recipe(
+            'registration.tests.utils.operation',
             operator=approved_user_operator.operator,
+            status=Operation.Statuses.REGISTERED,
             _quantity=20,
         )
+        for operation in registered_operations:
+            baker.make_recipe(
+                'registration.tests.utils.operation_designated_operator_timeline',
+                operation=operation,
+                operator=approved_user_operator.operator,
+                end_date=None,
+            )
```

Comment:
this is ugly in the sense that it's adding in more lines of code, but it's the only solution I could find to get around this error:

`django.core.exceptions.ValidationError: {'__all__': ['Constraint “unique_active_designated_operator_per_operation” is violated.']}`